common/sgx_workload/workload/workload_processor.h

```diff
@@ -91,6 +114,11 @@ class WorkloadProcessor {
 #define IMPL_WORKLOAD_PROCESSOR_CLONE(TYPE) \
    WorkloadProcessor* Clone() const { return new TYPE(*this); }
 
+#define GET_WORKLOAD_PROCESSOR(_1,_2,_3,NAME,...) NAME
+
+#define REGISTER_WORKLOAD_PROCESSOR(...) \
```

Comment:
Please add a comment that workloads are expected to use `REGISTER_WORKLOAD_PROCESSOR` macro. This macro will invoke appropriate macro based on the number of arguments

Reply:
Done